### DISCUSSION TITLE
怎么关闭entity的聚焦功能

### DISCUSSION THREAD

**Discussion by j524275214** [zouyaoji/vue-cesium#184] [Q&A] [answered]
我想把这个功能关闭了 该怎么做？
![image](https://user-images.githubusercontent.com/40748734/161942590-2762f962-ff4d-476f-89a3-01902e78799a.png)

**Comment by zouyaoji** [comment, marked as answer]
vc-viewer 组件上 `selectionIndicator` 和 `infoBox` 属性写为 false。

**Reply by j524275214** [reply]
非常感谢 我也很喜欢乔巴！
